### PR TITLE
feat: cleanup post card share experiment

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Cleanup post card share experiment

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-915 #done

### CODE REVIEW

File: packages/shared/src/components/post/common.tsx [hunk 1]
```diff
@@ -5,10 +5,7 @@ import { ArticleOnboardingVersion } from '../../lib/featureValues';
 
 export type PostCardTests = Pick<
   ActionButtonsProps,
-  | 'postCardVersion'
-  | 'postCardShareVersion'
-  | 'postModalByDefault'
-  | 'postEngagementNonClickable'
+  'postCardVersion' | 'postModalByDefault' | 'postEngagementNonClickable'
```
Comment (sshanzel) [inline packages/shared/src/components/post/common.tsx:8]: Now that I saw this. I think we should create a ticket for cleaning up these ones.

**Review by sshanzel — APPROVED** [review]
I always love reviewing cleanups. Thank you for taking this on! 🙇

File: packages/shared/src/components/post/common.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/post/common.tsx:8]: yeah, we are awaiting Sab's document on which ones can all be cleanup.
Until then very low priority unless we touch something as with this one 😂

**Review by el — APPROVED** [review]
Yayy, love seeing code being removed 💃

**Pull request merged**